parcels/numba/README.md

```diff
@@ -0,0 +1,49 @@
+# Numba for Parcels, a report
+
+This is a report for the Numba-Parcels project of the ITS-Parcels team. As a summary, we have investigated the feasibility of implementing a parcels version that runs on Numba, instead of JIT/Scipy. In the end we find that at this stage it is probably possible, but would still require a lot of work, while being much slower than JIT and less flexible than Scipy, with higher maintanance. The main difficulties arise because of Parcels design philosphy of allowing users a lot of leeway in defining types and variables in their simulation. While this of course is one of the main strengths of Parcels as a library, it does pose challenges when working with Numba.
```

Comment:
And I find the 'while being much slower' a bit confusing. I could change to 'And even if implemented, it would very likely still be much slower', which is more explicit